hack/testing/run_daemon_cri_e2e.sh

```diff
@@ -24,11 +24,11 @@ DEFAULT_SKIP+="|should execute prestop exec hook properly"
 DEFAULT_SKIP+="|should execute poststart exec hook properly"
 DEFAULT_SKIP+="|should function for intra-pod communication: http*"
 DEFAULT_SKIP+="|should function for intra-pod communication: udp*"
-DEFAULT_SKIP+="|should project all components that make up the projection API*"
 export SKIP=${SKIP:-${DEFAULT_SKIP}}
 
 # FOCUS focuses the test to run.
-export FOCUS=${FOCUS:-}
+DEFAULT_FOCUS="\[NodeConformance\]"
```

Comment:
Only `NodeConformance` tagged test should be tested.  See https://github.com/kubernetes/community/blob/master/contributors/devel/conformance-tests.md